backend/src/controllers/project_status/index.js

```diff
@@ -1,5 +1,5 @@
-const useController = require("./useController/index.js");
-const model = require("../models/project_status");
+const useController = require("../useController/index.js");
+const model = require("@models/project_status");
```

Comment:
Is this the final folder structure? It seems like this file for example could be in controllers/projects/status.js.

Reply:
good find, I have updated the folders.

Reply:
I thought we were grouping them the same way things are grouped in the frontend menu, something like this:
projects
	- change_request
	- client_coding
	- journal_voucher
contracts
	- amendments
	- contract_deliverables
	- contract_resources
	- invoices
	- invoice_deliverables
	- invoice_resources
admin
	- contacts
	- glossary
	- ministry
 	- resources
	- subcontractors
	- suppliers
	- users

Reply:
ok I redo it.